my-app/src/components/UserList/UserList.tsx

```diff
@@ -0,0 +1,17 @@
+import UserListProps from '../../types/UserListProps';
```

Comment:
No need separate file, put it in component to easier when  component props update.